docs/source/quickstart.rst

```diff
@@ -0,0 +1,125 @@
+As well as ordinal hyperparameters.
+
+Tools like `SMAC3`_, `BOHB`_ or `auto-sklearn`_ can sample hyperparameter-configurations from this ConfigSpace.
+
+This simple quickstart tutorial will show you, how to set up you own ConfigSpace, and what you can realize with it.
+To accomplish this task, we need to:
+
+- Create a ConfigSpace
+- Define hyperparameters and their value ranges
+- Add the hyperparameters to the ConfigSpace
+- (Optional) Add constraints to the ConfigSpace
+
+We will show those steps in an exemplary way, by creating a ConfigSpace for a soft-margin SVM classifier and a neural network.
+
+1. Example: Basic Usage and Conditional Hyperparameter
+------------------------------------------------------
+
+Assume, we want to train a soft-margin SVM classifier with a RBF-kernel. This classifier has at least two hyperparametrs.
+A regularization constant :math:`\mathcal{C}` and a kernel hyperparameter  :math:`\gamma` with
+
+- :math:`\mathcal{C} \in` {10, 100, 1000}
+- :math:`\gamma \in` {0.1, 0.2, 0.5, 1.0}
+
+| How this classifier is implemented is not necessary in this example and thus not shown.
+
+The first step is always to create a ConfigSpace-object. All the hyperparameters and constraints will be added to this
+object.
+::
+
+   import ConfigSpace as CS
+   cs = CS.ConfigurationSpace()
+
+Now, we have to define the hyperparameters :math:`\mathcal{C}` and :math:`\gamma`. Since each hyperparameter only can
+take on values from a given subset, we use categorical-hyperparameters in this case.
+:: 
+
+   import ConfigSpace.hyperparameters as CSH
+   c = CSH.CategoricalHyperparameter(name='C', choices=[10, 100, 1000])
+   gamma = CSH.CategoricalHyperparameter(name='gamma', choices=[0.1, 0.2, 0.5, 1.0])
+
+As last step in this first example, we only need to add them to the ConfigSpace::
+
+   cs.add_hyperparameter(c)
+   cs.add_hyperparameter(gamma)
+
+And that's it.
+The ConfigSpace *cs* stores the hyperparameters :math:`\mathcal{C}` and :math:`\gamma` with their defined value-ranges.
+
+2. Example: IntegerHyperparameters and FloatHyperparameters:
+------------------------------------------------------------
+
+| As already mentioned, ConfigSpace is also able to distinguish between some other common types like integer- and float-hyperparameter.
+| In the next example, we will create a ConfigSpace for a simple neural network.
+| Assume this neural network model has a float-hyperparameter *learning_rate* and a integer-hyperparameter
+  *number of units in the hidden layer*.
+| Since the learning rate should be sampled from a value range from 0.000001 to 0.1, we choose this hyperparameter to be
+  a float-hyperparameter.
+| Note, that the parameter *log* is set to True: This causes that the values of this hyperparameter
+  is sampled from a logarithmic scale.
+  In hyperparameter-optimization this is sometimes useful, because a logarithmic scale enables us to search a bigger space quickly.
```

Comment:
bigger space -> larger space

This is not correct. It basically states that a change from 0.01 to 0.1 is as important/influential as a change from 0.001 to 0.01.